### PR TITLE
Collaboration fragment implementation #13 solved

### PR DESCRIPTION
**Pull request by sdcode001**
https://user-images.githubusercontent.com/92887905/156877965-32b5ae60-9035-485f-9c93-06a22ab65cb1.mp4

isssu #13 
Collaboration fragment implementation successful implemented
All the layouts,Recycler views, Adapters,View models,models  created successful and placed at the suitable folders and UI is perfectly responsive .......here is the screenrecording

### CODE REVIEW

**Comment by HeySitam** [comment]
Great work bro. One simple thing you need to do. Can place the "List of Patents" text view in middle of yellow and blue background joint.

**Comment by sdcode001** [comment]
ok I'll do that

**Comment by sdcode001** [comment]
actually i don't getting  how to  place the "List of Patents" text view in middle of yellow and blue background joint. can you please help me?

**Comment by HeySitam** [comment]
Yeah sure bro. @chayan-dev will definitely help you out.

**Comment by sdcode001** [comment]
@chayan-dev help needed in this issue

**Comment by sdcode001** [comment]
@sitamadex11 how to resolve the conflict here?

**Comment by HeySitam** [comment]
I will handle it. Are you able to do the task? Else let me know whenever you get time. we can join in the voice channel.

**Comment by sdcode001** [comment]
ok sure. I'll ping you on discord

**Comment by cdhiraj40** [comment]
any updates @sdcode001 ?

**Comment by sdcode001** [comment]
trying

**Comment by sdcode001** [comment]
@sitamadex11 @cdhiraj40  I tried with so many different ways to place the  "list of  patents"  textview in between yellow and blue background joint as @cdhiraj40 suggested bt nothing worked at all. Do u guys have any other ideas to do that?

**Comment by HeySitam** [comment]
Bro for now I am merging. then once come to VC. I can show you how to do such. Let me know whenever you will be free.

File: app/src/main/res/layout/collaboration_recyclerview_item.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (HeySitam) [inline app/src/main/res/layout/collaboration_recyclerview_item.xml:1]: bro in every layout related XML files there should be `<?xml version="1.0" encoding="utf-8"?>`

File: app/src/main/java/com/gcect/gcectapp/viewmodels/PatentViewmodel.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package ViewModel
```
Comment (HeySitam) [inline app/src/main/java/com/gcect/gcectapp/viewmodels/PatentViewmodel.kt:1]: bro always recheck the package name before PR. coz this is simply crashing the application.

File: app/src/main/res/layout/fragment_collaboration_.xml [hunk 3]
```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
```
Comment (HeySitam) [inline app/src/main/res/layout/fragment_collaboration_.xml:6]: also avoid height and width in <layout> </layout> tag

**Comment by HeySitam** [comment]
@sdcode001  please add the .idea/misc.xml to git ignore.

**Comment by sdcode001** [comment]
@sitamadex11 how to put .idea/misc.xml into git ignore?

**Comment by HeySitam** [comment]
https://stackoverflow.com/questions/11968531/what-to-gitignore-from-the-idea-folder you can check this as refernce.

**Comment by HeySitam** [comment]
bro I have resolved all the conflicts. still your .idea/misc.xml is occurring some problem
And also it's best practice not to push origin .idea

**Comment by sdcode001** [comment]
okk trying

**Comment by sdcode001** [comment]
@sitamadex11 can I again make the pr for the same. with idea/misc.xml at gitignore.

**Comment by sdcode001** [comment]
bro @sitamadex11 what to do now?

**Comment by sdcode001** [comment]
@sitamadex11 i have added .idea/misc.xml to .gitignore file but the conflict comes again.

**Comment by sdcode001** [comment]
@cdhiraj40 @chayan-dev can you guys please check it............ I have added .idea/misc.xml to the .gitignore file but the conflict comes again.

**Comment by sdcode001** [comment]
@sitamadex11  please do something with this PR

**Comment by cdhiraj40** [comment]
> @cdhiraj40 @chayan-dev can you guys please check it............ I have added .idea/misc.xml to the .gitignore file but the conflict comes again.

hey @sdcode001 the conflicts still come as those files were already committed, ``.gitignore`` wont consider the next changes. you can create new PR where you add ``.idea`` and ``.misc`` to .gitignore before you start working!

**Pull request closed without merging**